src/consumption.cpp

```diff
@@ -52,6 +52,9 @@ const std::vector<std::string> carnivore_blacklist {{
 const std::array<std::string, 2> temparray {{"ALLERGEN_MEAT", "ALLERGEN_EGG"}};
 const std::vector<std::string> herbivore_blacklist( temparray.begin(), temparray.end() );
 
+//The maximum amount of mass and volume that a internal furnace can consume
+const units::volume max_volume( 3000, units::volume_in_milliliter_tag{} );
```

Comment:
You can write `3000_ml` when you need an instance of `units::volume` (it's called "user-defined literals").

Reply:
thank you, changed the name to furnace_max_volume